modules/sonobiBidAdapter.js

```diff
@@ -141,6 +141,18 @@ var SonobiAdapter = function SonobiAdapter() {
     return 'desktop';
   }
 
+  function _getReferrer(bids) {
```

Comment:
there's a utility called getTopFrameReferrer in the util lib you can use here.

https://github.com/prebid/Prebid.js/blob/master/src/utils.js#L175